docs/under-the-hood/storage.md

```diff
@@ -1,138 +1,201 @@
+
+$19148 \times (112\text{B} + 12\text{B} + 16\text{B} + 16\text{B} + 104\text{B}) = 19148 \times 260\text{B} = 4978480\text{B}.$
+
+The remaining 2584 vertices are the `ComicSeries` vertices with the `title` and `publishYear` properties. The `publishYear` property is a list of integers. The average length of the `publishYear` list is 2.17, so to be sure, we'll say that each list has 3 elements. Since the integer is the year, 2B for each integer will be more than enough. Therefore, each list occupies $3 \times 2\text{B} = 6\text{B}$. We are going to use the same formula as above, we just have to be careful to include both `title` and `publishYear` properties. We can assume that the `title` property is approximately the same length as the `name` property. We have:
+
+$2584 \times (112\text{B} + 12\text{B} + 6\text{B} + 16\text{B} + 16\text{B} + 104\text{B}) = 2584 \times 266\text{B} = 687344\text{B}.$
+
+
+In total, $5665824\text{B}$ to store vertices.
+
+The edges don't have any properties on them, so the following formula is:
+
+$\texttt{NumberOfEdges} \times (\texttt{Edge} + \texttt{SkipListNode} + \texttt{next\_pointers} + \texttt{Delta}).$
+
+There are 682943 edges in the Marvel dataset. Hence, we have:
+
+$682943 \times (40\text{B}+16\text{B}+16\text{B}+104\text{B}) = 682943 \times 176\text{B} = 120197968\text{B}.$
+
+Next, we have label index on `Hero`, `Comic` and `ComicSeries` labels. To calculate how much space they take, we can use the following formula:
+
+$\texttt{NumberOfLabelIndices} \times \texttt{NumberOfVertices} \times (\texttt{SkipListNode<LabelIndex::Entry>} + \texttt{next\_pointers})$
+
+Since there are three label indices, we have the following calculation:
+
+$3 \times 21723 \times (24\text{B}+16\text{B}) = 65169 \times 40\text{B} = 2606760\text{B}.$
+
+For label-property index, we have to take labeled property into account. Property `name` is indexed on `Hero` and `Comic` vertices, while property `title` is indexed on `ComicSeries` vertices. We already assumed that the `title` property is approximately the same length as the `name` property. 
+
+Here is the formula:
+
+$\texttt{NumberOfLabelPropertyIndices} \times \texttt{NumberOfVertices} \times (\texttt{SkipListNode<LabelIndex::Entry>} + \texttt{property} + \texttt{next\_pointers}),$
+
+and when we include the appropriate values, we get:
 
-### Disabling properties on relationships
+$3 \times 21723 \times (80\text{B}+12\text{B}+16\text{B})= 65169 \times 108\text{B} = 7038252\text{B}.$
 
-If you have a use-case that doesn't use properties on relationships, you can
-specify a flag in the Memgraph configuration file to disable them and reduce
-memory usage.
+Now let's sum up everything we calculated:
 
-```
---storage-properties-on-edges=false
-```
+$5665824\text{B} + 120197968\text{B} + 2606760\text{B} + 7038252\text{B} = 135508804 \text{B} \approx 135\text{MB}.$
```

Comment:
@katarinasupe It is nice that you got into 10MB-s range from initial approximations. I like this style of example as demo, but maybe when we go distributed or have extra time, we should rewrite it with something simpler than Marvel, this is a bit complex and lengthy. But overall great effort, and well done! 💪

Reply:
Thanks and I agree it can be done on an even simpler example.